### PR TITLE
modernize

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/zendesk-rails-upgraders

### CODE REVIEW

**Comment by grosser** [comment]
tried that before and it ended up installing old versions like 2.4.1

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,12 @@ language: ruby
...
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.3
-  - 2.3.0
-  - 2.4.1
+  - 2.2.9
+  - 2.3.4
+  - 2.4.3
+  - 2.5.0
```
[the remark this answers is not in the document]
Comment (bquorning) [reply .travis.yml]: > tried that before and it ended up installing old versions like 2.4.1

Sometimes I wonder why we even use Travis…

[the remark this answers is not in the document]
Comment (grosser) [reply .travis.yml]: I'll try again and open a ticket if it's broken :)

**Comment by grosser** [comment]
here is 2.2.9 which took 20s to install 
https://travis-ci.org/zendesk/stronger_parameters/jobs/357643141

and there is 2.2 which too 4s to install 2.2.7
https://travis-ci.org/zendesk/stronger_parameters/jobs/357803470

 ... so I'd rather send the extra time :)
opened an issue to always get latest though

**Comment by grosser** [comment]
should be all green now ... looks good ?

**Comment by grosser** [comment]
bumping to 2.5 revealed existing coverage gaps, so needed more tests and mocha to reach those edge-cases
